vhdbuilder/packer/install-dependencies.sh

```diff
@@ -107,7 +107,9 @@ if [[ $(isARM64) == 1 ]]; then
   fi
 fi
 
-if [[ "${UBUNTU_RELEASE}" == "18.04" || "${UBUNTU_RELEASE}" == "20.04" || "${UBUNTU_RELEASE}" == "22.04" ]]; then
```

Comment:
@UtheMan its fine to do it like this here because this is VHD building code, and we don't build 1604 VHDs anymore, but CSE is still run for 1604 so cant make a similar change there